include/action.hpp

```diff
@@ -0,0 +1,14 @@
+#ifndef MEOWCHKIN_ACTION_HPP
+#define MEOWCHKIN_ACTION_HPP
+
+enum class some_commands {};  // will be specified in future
+
+namespace meow {
+class Action {
+ public:
+  std::vector<some_commands> commands;
+  std::optional<std::thread::id> target;
```

Comment:
what is target? come up with a better name or write a comment to explain what does it mean